src/redux/configureStore.js

```diff
@@ -0,0 +1,14 @@
+import { combineReducers, createStore, applyMiddleware } from 'redux';
+import logger from 'redux-logger';
+import booksReducer from './books/books';
+
+const reducer = combineReducers({
+  booksReducer,
+});
+
+const store = createStore(
+  reducer,
+  applyMiddleware(logger),
+);
```

Comment:
- [ ] Kindly add categories in configureStore.js, the way you have added books with combining reducers.